Electron/src/components/Sidebar/ModalAddSongPlaylist/Accordion/AddSongPlayListAccordion.tsx

```diff
@@ -409,12 +417,18 @@ export default function AddSongPlayListAccordion({
                   />
                 </div>
               </div>
+              <p className="text-danger mt-2">
+                Los campos marcados con * son obligatorios.
+              </p>
 
               <button
                 type="button"
                 onClick={handleSubmitSong}
                 className={`btn btn-lg ${styles.btnSend} d-flex flex-row justify-content-center`}
                 data-testid="sidebar-addsongplaylistaccordion-submit-song"
+                disabled={
+                  !formDataSong.name && !formDataSong.genre && !songFile
```

Comment:
I think this has to be an or statement, we want all the fields to be included

Reply:
> I think this has to be an or statement, we want all the fields to be included. Tests would be good for this case

yes this are all mandatory field, button will be disabled until and unless user fill all the required field. or may be you wanna say something else which i could not understand please elaborate.

Reply:
I checkout your branch to test if it was working correctly. Playlist form worked fine but in form song I found the following:

* I can create a song with ONLY 1 required field filled. We want ALL required fields to be filled in order to enable the button and allow the user to upload a song.
* The disabled condition on that form relies on 'and' statements. The field will be disabled if name field is not filled and song field is not filled and so. We want the logic to be: if name field is not filled or song file is not filled or genre is not selected....

TL;DR -> Disabled logic for song form has to be inversed. Right now you can upload a song with required fields not filled

Reply:
so thats what the logic is is if name, genre, song are filled then only submit button will be enabled. or may be you are trying to say that among name, genre, song only one field should be filled and then button should be enabled right ? so should i convert it from and to or statement

Reply:
All required fields have to filled so the button appears enable

Negating that condition we get :
* Any required field that is not filled should make the button disabled. Your code does exactly the opposite. Launch backend and frontend and test the behaviour of song form, with your code it will let you submit a song with only one required field filled and we don't want that

Reply:
ok i understood now i have added or condition now button will enabled when all field are present which are required. i checked form from my side as well